<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightTestWidget</class>
 <widget class="QWidget" name="FlightTestWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1049</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>460</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>460</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0" colspan="4">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="mLaunchTestButton">
          <property name="text">
           <string>Perform Flight Test</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="mMoverLabel">
          <property name="text">
           <string>Mover</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="mMoverComboBox"/>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <widget class="QGroupBox" name="mSettingsGroupBox">
        <property name="title">
         <string>Initial Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="3" column="1">
          <widget class="QLineEdit" name="mHeadingLineEdit">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="mSpeedLineEdit">
           <property name="text">
            <string>750</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="mAltitudeLabel">
           <property name="text">
            <string>Altitude</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="mLatitudeLineEdit">
           <property name="text">
            <string>21.12</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="mLongitudeLineEdit">
           <property name="text">
            <string>-157.95</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="mSpeedLabel">
           <property name="text">
            <string>Speed</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="mLatitudeLabel">
           <property name="text">
            <string>Latitude</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="mHeadingLabel">
           <property name="text">
            <string>Heading</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="mLongitudeLabel">
           <property name="text">
            <string>Longitude</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="mAltitudeLineEdit">
           <property name="text">
            <string>20000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>feet</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>deg</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>ft/sec</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="1" column="0">
       <widget class="QPushButton" name="mPreviousButton">
        <property name="text">
         <string>Previous</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="Designer::AnimatedGeometryGLWidget" name="mAnimatedGLWidget"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Designer::AnimatedGeometryGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>AnimatedGeometryGLWidget.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mLatitudeLineEdit</tabstop>
  <tabstop>mLongitudeLineEdit</tabstop>
  <tabstop>mAltitudeLineEdit</tabstop>
  <tabstop>mHeadingLineEdit</tabstop>
  <tabstop>mSpeedLineEdit</tabstop>
  <tabstop>mLaunchTestButton</tabstop>
  <tabstop>mPreviousButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
